scylla/src/lib.rs

```diff
@@ -163,10 +163,6 @@ pub mod serialize {
             BuiltinSerializationError, BuiltinSerializationErrorKind, BuiltinTypeCheckError,
             BuiltinTypeCheckErrorKind,
         };
-
-        // Not part of the old framework, but something that we should
-        // still aim to remove from public API.
-        pub use scylla_cql::serialize::row::{SerializedValues, SerializedValuesIterator};
```

Comment:
:question: Aren't these used by benchmarks?

Reply:
Benchmarks import them using `scylla_cql::` path instead of those re-exports.